docs/contrib/ides.rst

```diff
@@ -0,0 +1,20 @@
+.. _ides:
+
+Be sure to execute ``sbt rebuild`` so that artifacts are published and the IDEs can resolve them.
```

Comment:
After the recent changes in the build, this should not be necessary anymore.

Reply:
:heavy_check_mark: